cmake/Modules/ArrowExternalProject.cmake

```diff
@@ -92,7 +87,7 @@ endif()
 
 ExternalProject_Add(arrow_ep
   PREFIX external/arrow
-  DEPENDS flatbuffers boost
+  DEPENDS flatbuffers boost glog
```

Comment:
Does this cause Arrow to use the version of `glog` compiled by Ray? Or will Arrow clone/build its own copy?

Reply:
I am taking national day in China. I can test it on centos after Friday. I added this just for including @yuhong’ pr, em... I think it needs to pass the GLOG_HOME to arrow cmake args.